modern/src/components/registration/LoginForm.js

```diff
@@ -124,8 +129,8 @@ const LoginForm = () => {
           <Grid item xs>
             <FormControl variant="filled" fullWidth>
               <InputLabel>{t('loginLanguage')}</InputLabel>
-              <Select>
-                <MenuItem value="en">English</MenuItem>
+              <Select value={language} onChange={handleLanguageChange}>
+                {languageList.map((lang) => <MenuItem key={lang.code} value={lang.code}>{lang.name}</MenuItem>)}
```

Comment:
Please also don't use shortened words for variable. If you want something short here, you can just use `it` if the meaning is obvious.

Reply:
Yes sure.

Reply:
> Please also don't use shortened words for variable. If you want something short here, you can just use `it` if the meaning is obvious.

You mean I can use the word ```it``` instead of ```lang``` ?

Reply:
Yes, I would recommend either full `language` or `it`.